docs/source/configuration.rst

```diff
@@ -413,6 +414,8 @@ and the required credentials:
   * ``userserv``: the service's nickame to send credentials to
     (``UserServ`` by default)
 
+Click a directive name above for more details about the available options.
```

Comment:
You'll confuse the hell out of beginners who'll try to understand what a "directive" is. Also, most people will stop their reading at the list, I'd suggest to update the line *before*:

> These additional options can be used to configure the authentication method
> and the required credentials (you can follow the link on each option for more details):

Or something similar.